core/options.js

```diff
@@ -155,6 +155,12 @@ Blockly.Options = function(options) {
  */
 Blockly.BlocklyOptions = function() {};
 
+/**
+ * The SVG element for the grid pattern.
+ * @type {SVGElement}
+ */
+Blockly.Options.prototype.gridPattern = null;
```

Comment:
Can we move this to the constructor.

Reply:
Even if it's not passed into the options and you just want to set it to ``null``. I would set it in the constructor as per https://github.com/google/blockly/issues/2892

Reply:
Updated.